scripts/SasLowNoisePolicy.psm1

```
@@ -35,12 +35,135 @@ function Get-SasCanonicalLowNoiseDocument {
+        FallbackDecisionNames = @(
+            'default_ok',
+            'web_only_fallback',
+            'approved_subnet_host_discovery_required',
+            'udp_justification_required',
+            'all_ports_denied_without_explicit_gate',
+            'review_required'
+        )
+    }
+}
+
+function Get-SasPortFallbackDecision {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-SasPortFallbackDecision' does not have a help comment.

(PSProvideCommentHelp)